validate/validate_test.go

```diff
@@ -656,6 +656,13 @@ func TestCheckPlatform(t *testing.T) {
 			platform: "windows",
 			expected: specerror.NonError,
 		},
+		{
+			val: rspec.Spec{
+				Version: "1.0.0",
```

Comment:
@zhouhao3 can you please change this to 1.0.2 (which is currently the minimum version required (as explained in commit 67884fc9e3b1a61a85db024c0cf33710e57cb37f)?

Reply:
@zhouhao3 ^^^